### PR TITLE
Loadout builder -- now with slightly better mobile styling!

### LINKED ISSUE #3522: (Minor) 2 Perks in Loadout Builder Pushes UI Down
cbe5f081-b890-4bf6-a50c-24e1da22c885
Selecting 2 Locked Perks in Loadout Builder, pushes the rest of the Lock Perk / Item UI down.

Browser - Safari (Mobile)
OS - iOS 12
Device - iPhone 7+

### LINKED ISSUE #3541: Loadout builder visually identify which finder is selected
Loadout builder has no visual for ammo finders after they have been selected
screenshot attached has heavy and special finders ##
image

### PR DESCRIPTION
**Pull request by kyleshay**
Just css updates, resolves #3522

rec2

also resolves #3541
screen shot 2019-02-07 at 10 26 06 pm

### CODE REVIEW

**Comment by duckmanBR** [comment]
I know this update is just CSS, but could we fit in a manual update button/selector?  Because on mobile (at least) I need to wait some seconds to the build generator finishes so I can select another perk.

Also, the icons are huge like this because of your column settings, right?

**Comment by kyleshay** [comment]
Ah, yeah adding a setting to recalculate automatically or not, and then having a button would be a nice idea. 

I think the icons are just larger on mobile. It could be related to my phone/DIM settings

File: src/app/dim-ui/BungieImage.tsx [hunk 1]
```diff
@@ -15,8 +18,18 @@ interface BungieImageProps {
...
-  return <img src={bungieNetPath(src)} {...otherProps} />;
+  const { src, hash, ...otherProps } = props;
+
+  let styles = props.className;
+  if (hash) {
+    styles = classNames(props.className, 'perk-image', {
+      'ammo-primary': hash === 143442373,
+      'ammo-special': hash === 2620835322,
+      'ammo-heavy': hash === 2867719094
+    });
```
Comment (bhollis) [inline src/app/dim-ui/BungieImage.tsx]: I'd prefer not to add this sort of thing into `BungieImage` - better to make a wrapped component.

File: src/app/dim-ui/BungieImage.tsx [hunk 2]
```diff
@@ -16,6 +16,7 @@ export default function BungieImage(
   props: BungieImageProps & React.ImgHTMLAttributes<HTMLImageElement>
 ) {
   const { src, ...otherProps } = props;
+
```
Comment (kyleshay) [inline src/app/dim-ui/BungieImage.tsx]:
```suggestion
```

File: src/app/dim-ui/StyledBungieImage.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+interface BungieImageProps {
+  src: BungieImagePath;
+  hash: number;
+}
+
+/**
+ * A styled BungieImage that overlays additional meta-information.
+ */
+export default function StyledBungieImage(
```
Comment (bhollis) [inline src/app/dim-ui/StyledBungieImage.tsx]: Can we call this BungieImageAndAmmo or something?

File: src/app/loadout/loadout-popup.scss [hunk 4]
```diff
@@ -62,18 +62,6 @@
       }
     }
   }
-
```
Comment (duckmanBR) [inline src/app/loadout/loadout-popup.scss]: This may be the cause of #3618

**Pull request merged**